src/options_app.c

```diff
@@ -553,7 +436,12 @@ BOOL OptionsApp_Exit(OVY_MANAGER *manager, int *state) {
 }
 
 static void OptionsApp_SetupGraphicsBanks(void) {
-    GraphicsBanks banks = sOptionsAppGraphicsBanks;
+    GraphicsBanks banks = {
```

Comment:
this was fine before, pls revert

Reply:
no it was not, it will remain as-is

Reply:
bruh

Reply:
I can accept going back through the new stuff to change myself, but old stuff should be kept the same